app/Resources/views/modules/footer.html.twig

```diff
@@ -0,0 +1,36 @@
+<footer class=" nav navbar-fixed-bottom footer">
+    <div class="container-fluid">
+        <div class="row box row-xs-center">
+            <div class="col-sm-2 col-xs-12 footer-menu--logoMobile">
+                <a class= "footer-link" href=""><img class="footer-img" src="{{ asset('img/logo-pin.png') }}"></a>
+            </div>
+            <div class="col-xs-12 hr--hidden">
+                <hr>
+            </div>
+            <div class="col-sm-2 col-sm-offset-1 col-xs-12 footer-menu">
+                <a class="footer-link" href="#">DEVENIR RESTAURATEUR</a>
```

Comment:
Here and for the other texts, we need translations :) something like {{ 'placeholder' |trans({}, 'name_of_translation_file') }}

You can generate translations with `app/console translations:update --force --output-format=xlf fr`